README.md

```diff
@@ -0,0 +1,141 @@
+# talk
```

Comment:
Talk with a big T?

Reply:
I typically place the first line equal to the repository name, which in this case, is lowercased.